src/osd/scheduler/OpSchedulerItem.h

```diff
@@ -156,6 +169,16 @@ class OpSchedulerItem {
   utime_t get_start_time() const { return start_time; }
   uint64_t get_owner() const { return owner; }
   epoch_t get_map_epoch() const { return map_epoch; }
+  dmc::ReqParams get_qos_req_params() const { return qos_req_params; }
+  void set_qos_req_params(dmc::ReqParams qparams) {
```

Comment:
Does this actually have a user?  Looks like the constructor is the only place this gets set.

Reply:
Currently there are no users and therefore I removed it for now.